Azure Service Bus Updates (#5911)
Particular/docs.particular.net: transports/azure-service-bus/configuration_lockrenewal_asbs_[3,).partial.md

> It is possible to renew the lock automatically for longer than the default by overriding the `MaxAutoLockRenewalDuration`.

Reviewer: ```suggestion
The lock can be auto-renewed beyond the default by overriding the `MaxAutoLockRenewalDuration`.
```
